yt_dlp/extractor/gettr.py

```diff
@@ -50,43 +51,70 @@ class GettrIE(GettrBaseIE):
             'ext': 'mp4',
             'uploader': 'Neues Forum Freiheit',
             'uploader_id': 'nf_freiheit',
+            'upload_date': '20210718',
             'thumbnail': r're:^https?://.+/out\.jpg',
-            'timestamp': 1626594455017,
+            'timestamp': 1626594455.017,
             'duration': 23,
             'tags': 'count:12',
         }
+    }, {
+        # quote post
+        'url': 'https://gettr.com/post/pxn5b743a9',
+        'only_matching': True,
+    }, {
+        # quote with video
+        'url': 'https://gettr.com/post/pxtiiz5ca2',
+        'only_matching': True,
+    }, {
+        # streaming embed
+        'url': 'https://gettr.com/post/pxlu8p3b13',
+        'only_matching': True,
+    }, {
+        # youtube embed
+        'url': 'https://gettr.com/post/pv6wp9e24c',
+        'only_matching': True,
+        'add_ie': ['Youtube'],
     }]
 
     def _real_extract(self, url):
         post_id = self._match_id(url)
         webpage = self._download_webpage(url, post_id)
-
         api_data = self._call_api('post/%s?incl="poststats|userinfo"' % post_id, post_id)
 
         post_data = api_data.get('data')
-        user_data = try_get(api_data, lambda x: x['aux']['uinf'][post_data['uid']]) or {}
+        user_data = try_get(api_data, lambda x: x['aux']['uinf'][post_data['uid']], dict) or {}
 
-        if post_data.get('nfound'):
-            raise ExtractorError(post_data.get('txt'), expected=True)
+        vid = post_data.get('vid')
+        ovid = post_data.get('ovid')
+
+        if post_data.get('p_type') == 'stream':
+            return self.url_result(f'https://gettr.com/streaming/{post_id}', ie='GettrStreaming', video_id=post_id)
+
+        if not (ovid or vid):
+            embed_url = url_or_none(post_data.get('prevsrc'))
+            shared_post_id = traverse_obj(api_data, ('aux', 'shrdpst', '_id'), ('data', 'rpstIds', 0), expected_type=str)
+
+            if embed_url:
+                return self.url_result(embed_url)
+            elif shared_post_id:
+                return self.url_result(f'https://gettr.com/post/{shared_post_id}', ie='Gettr', video_id=shared_post_id)
+            else:
+                raise ExtractorError('There\'s no video in this post.')
 
         title = description = str_or_none(
-            post_data.get('txt') or self._og_search_description(webpage))
+            post_data.get('txt')
+            or self._og_search_description(webpage))
 
         uploader = str_or_none(
             user_data.get('nickname')
-            or remove_end(self._og_search_title(webpage), ' on GETTR'))
+            or self._search_regex(r'^(.+?) on GETTR', self._og_search_title(webpage, default=''), 'uploader', fatal=False))
```

Comment:
example where old code fails?

Reply:
They changed the og:title to include the body of the post.
`<meta property="og:title" content="EpochTV on GETTR : Ethiopia and the ...">`